crates/typst/src/visualize/polygon.rs

```diff
@@ -32,11 +32,12 @@ pub struct PolygonElem {
     ///
     /// When setting a fill, the default stroke disappears. To create a
     /// rectangle with both fill and stroke, you have to configure both.
-    ///
-    /// Currently all polygons are filled according to the
-    /// [non-zero winding rule](https://en.wikipedia.org/wiki/Nonzero-rule).
     pub fill: Option<Paint>,
 
+    /// The rule used to fill the polygon.
+    /// Defaults to `non-zero`.
+    pub fill_rule: Smart<FillRule>,
```

Comment:
I don't think this needs to be `Smart`. It can instead just provide a default value (probably non-zero)